src/components/github-htmlview.component.js

```diff
@@ -344,7 +348,7 @@ export class GithubHtmlView extends Component {
           _index,
           _siblings,
           _parent,
-          _defaultRenderer,
+          _defaultRenderer
```

Comment:
@machour It seems your IDE configurations are not consistent with our ESLint rules. We require tail commas as I know.

Reply:
Interesting. Prettier should run on commit and fix all these, though.

Reply:
Yeah our prettier pre-commit hook should be taking care of this :thinking:

Reply:
Sorry for that guys, I tried to enhance my VSCode settings yesterday, this is related.

But something I don't quiet understand about prettier, is that it always seems to modify the files **after** my commit. 

Is it because I don't use `git add` in my workflow, but instead I do a `git commit` passing paths to the files I changed?

Reply:
We have [precommit script](https://github.com/gitpoint/git-point/blob/master/package.json#L21), whose details are [here](https://github.com/gitpoint/git-point/blob/master/package.json#L41). Once you want to commit something, prettier will modify your codes <strong>first</strong>, then finish the commit process.

It does't matter whether you use `git add` or not. There will be always a `git add` in the precommit script.

Reply:
At me by the way have ceased to work git hooks, what it is necessary to make, that they again earned?